### PR TITLE
Fix spelling/grammar in pedia.

### PR DESCRIPTION
**Pull request by aschuch247**

### CODE REVIEW

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -12742,7 +12742,7 @@ BLD_GATEWAY_VOID
 Gateway to the Void
 
 BLD_GATEWAY_VOID_DESC
-'''The ability to close off a system almost entirely can be useful for an empire on the defensive. Any ships who enter this system without passing through on the same turn are destroyed, and all colonies within are immediately reduced to [[encyclopedia OUTPOSTS_TITLE]] status. The [[metertype METER_STEALTH]] of all objects within the system is increased by 1000, making them almost impossible to detect.
+'''The ability to close off a system almost entirely can be useful for an empire on the defensive. Any ships which enter this system without passing through on the same turn are destroyed, and all colonies within are immediately reduced to [[encyclopedia OUTPOSTS_TITLE]] status. The [[metertype METER_STEALTH]] of all objects within the system is increased by 1000, making them almost impossible to detect.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: Should be `that` I think

Comment (aschuch247) [reply default/stringtables/en.txt]: For persons `who`, for things `which` and if you do not care or are not sure, `that`. Using `which` communicates that you are not talking about a person, but about a thing.

Comment (aschuch247) [reply default/stringtables/en.txt]: https://www.grammarly.com/blog/which-vs-that/

Well, it is a defining clause, so yes, `that` is the correct choice.

Comment (geoffthemedio) [reply default/stringtables/en.txt]: yes; I do not prefer "who" over "which".

**Pull request merged**